client.go

```
@@ -0,0 +1,48 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"io"
+	"net"
+	"os"
+	"strings"
+)
+
+func main() {
+	connection, _ := net.Dial("tcp", config.address+":"+config.port)
```

Please do not ignore an error returned by `net.Dial()`.